docs/how-to/update.md

```
@@ -0,0 +1,43 @@
+    uv tool upgrade untether
+    ```
+
+=== "pipx"
+
+    ```sh
+    pipx upgrade untether
+    ```
+
+Check your current version:
+
+```sh
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)